crates/ursa-network/src/behaviour.rs

```diff
@@ -159,10 +164,40 @@ impl<P: StoreParams> Behaviour<P> {
                 if relay_client.is_none() {
                     panic!("relay client not instantiated");
                 }
-                dcutr::behaviour::Behaviour::new()
+                Dcutr::new()
             })
             .into();
 
+        let mdns = config
```

Comment:
Revert

Reply:
Done. But code is now inconsistent

![image](https://user-images.githubusercontent.com/8976745/209259213-77f73140-68e5-4d6b-b1a3-da9d243cf622.png)